src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataDeltaFeedSerializer.cs

```diff
@@ -527,7 +538,7 @@ public async Task WriteDeltaLinkAsync(object graph, ODataWriter writer, ODataSer
             }
         }
 
-        private ODataDeletedResource GetDeletedResource(object graph)
+        private ODataDeletedResource GetDeletedResource(object graph, ResourceContext resourceContext)
         {
             EdmDeltaDeletedEntityObject edmDeltaDeletedEntity = graph as EdmDeltaDeletedEntityObject;
```

Comment:
>graph as EdmDeltaDeletedEntityObject; [](start = 64, length = 37)

Here, it only supports the typeless scenario,  Since we have the typed class,  DeltaDeletedEntityObjectOfT, we should support typed scenario.